_includes/footer.md

```diff
@@ -4,10 +4,7 @@
 [![Apache Software Foundation][ASF_IMG]{: #asf-logo height="100"}][ASF]
 
 Copyright &copy; 2011-{{ site.time | date: '%Y' }} The Apache Software
-Foundation, Licensed under the [Apache License, Version 2.0][AL2]. Apache
```

Comment:
I was trying to reduce the legalese in the footer as its on every page.  However, I see your point that it should be on the website somewhere.  I pushed another commit ( ff8f675615 ) where I created a notice page.  This is now linked to in the footer.  Any additional legal notices can be added here rather than putting them in the footer.  Does that work?